src/services/dumper/templates/agent-nodejs/index.hbs

```diff
@@ -0,0 +1,20 @@
+const dotenv = require('dotenv');
+dotenv.config();
+
+const { createAgent } = require('@forestadmin/agent');
+{{datasourceImport}}
+
+(async () => {
+  await createAgent({
+    authSecret: process.env.FOREST_AUTH_SECRET,
+    envSecret: process.env.FOREST_ENV_SECRET,
+    isProduction: process.env.NODE_ENV === 'production',
+    {{#if forestUrl}}
+    forestServerUrl: process.env.FOREST_SERVER_URL,
+    {{/if}}
+    typingsPath: './typings.d.ts',
+  })
+  .addDataSource({{datasourceCreation}})
+  .mountOnStandaloneServer(process.env.APPLICATION_PORT)
+  .start();
+})();
```

Comment:
```suggestion
require('dotenv').config();
const { createAgent } = require('@forestadmin/agent');
const { createSqlDataSource } = require('@forestadmin/datasource-sql');

// Create the Forest Admin agent.
/**
 * @type {import('@forestadmin/agent').Agent<import('./typings').Schema>}
 */
const agent = createAgent({
  authSecret: process.env.FOREST_AUTH_SECRET,
  envSecret: process.env.FOREST_ENV_SECRET,
  isProduction: process.env.NODE_ENV === 'production',
  {{#if forestServerUrl}}
  forestServerUrl: process.env.FOREST_SERVER_URL,
  {{/if}}
  // Autocompletion of collection names and fields
  typingsPath: './typings.ts',
})
  // Connect your datasources.
  .addDataSource(createSqlDataSource(
    {
      uri: process.env.DATABASE_URL,
      ssl: process.env.DATABASE_SSL,
      schema: process.env.DATABASE_SCHEMA || 'public',
    }));

    // Add customizations here.

  // Expose an HTTP endpoint.
  agent
    .mountOnStandaloneServer(Number(process.env.APPLICATION_PORT) || 3351, process.env.APPLICATION_URL)
    // Start the agent.
    .start();
```

Adapt to support mongoose

Reply:
Import should be dynamic
NODE_ENV is actually missing from the .env file, to be added
Let's consider the proposal on the sql datasource, but keep the dynamic behaviour to support mongo
APPLICATION_PORT is already computed with the correct value, and APPLICATION_URL is not required anymore.